lib/utils.js

```diff
@@ -162,7 +177,9 @@ function unsetConfig(options, config) {
     return obj
   }, {})
   const newValues = (config ? Object.keys(config) : [])
-  const diff = oldValues.filter(value => newValues.indexOf(value) < 0).filter(value => value)
+  const diff = oldValues.filter((value) => {
+    return newValues.indexOf(value) < 0
+  }).filter((value) => { return value })
```

Comment:
Could also potentially be `.filter(Boolean)` if we are just trying to remove falsy values